### ISSUE TITLE
QuickBooksOnline\API\Data\IPPWrapper class is incompatible with Composer 2.0+

### ISSUE THREAD

**Issue by JakeBooher** [intuit/QuickBooks-V3-PHP-SDK#377]
```
Class QuickBooksOnline\API\Data\IPPWrapper located in ./vendor/quickbooks/v3-php-sdk/src/Data/IPP.php does not comply with psr-4 autoloading standard. Skipping.
```

Renaming `src/Data/IPP.php` to `src/Data/IPPWrapper.php` should resolve this issue.

**Comment by JakeBooher** [comment]
Looks like someone has submitted a PR for this already in #375

**Comment by abisalehalliprasan** [comment]
@jacobembree: Thanks for reporting, this should be fixed with the release: [6.0.0](https://github.com/intuit/QuickBooks-V3-PHP-SDK/tree/6.0.0)
